test/powershell/Modules/Microsoft.PowerShell.Utility/Default-Aliases.Tests.ps1

```diff
@@ -27,7 +27,6 @@ Describe "Verify approved aliases list" -Tags "CI" {
         $FullCLR = !$isCoreCLR
         $CoreWindows = $isCoreCLR -and $IsWindows
         $CoreUnix = $isCoreCLR -and !$IsWindows
-        $FullCLR -or $CoreWindows -or $CoreUnix
         $aliasFullList = @{
```

Comment:
> "nwsn -> New-PSWorkflowSession"           =             $FullCLR

I don't see this alias on windows powershell ...
```
PS:4> Get-Alias -Definition New-PSWorkflowSession
Get-Alias : This command cannot find a matching alias because an alias with the definition 'New-PSWorkflowSession'
does not exist.
At line:1 char:1
+ Get-Alias -Definition New-PSWorkflowSession
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (New-PSWorkflowSession:String) [Get-Alias], ItemNotFoundException
    + FullyQualifiedErrorId : ItemNotFoundException,Microsoft.PowerShell.Commands.GetAliasCommand
```
Environment
```
Name                           Value
----                           -----
PSVersion                      5.1.14393.1066
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.14393.1066
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
```